packages/flutter/test/foundation/capabilities_test.dart

```diff
@@ -0,0 +1,12 @@
+// Copyright 2014 The Flutter Authors. All rights reserved.
+// Use of this source code is governed by a BSD-style license that can be
+// found in the LICENSE file.
+
+import 'package:flutter/foundation.dart';
+import 'package:flutter_test/flutter_test.dart';
+
+void main() {
+  test('isCanvasKit is true for CanvasKit', () {
+    expect(isCanvasKit, false);
+  }, skip: isCanvasKit); // [intended] isCanvasKit is what we are testing here.
```

Comment:
This test has no effect. Let's say `isCanvasKit` returns a wrong value. For example, we are in CanvasKit mode but the bool returns false. Then the test will run and `expect(isCanvasKit, false)` will pass. The opposite is the same. Therefore the test does not actually test anything.

Reply:
ah, that makes sense. I copied the `kIsWeb` test without much thought...

Reply:
Haha, maybe `kIsWeb` isn't testing anything either :) Perhaps what this test could test is that `isCanvasKit` doesn't crash. Correctness can be covered by the integration test.